x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/edit_job_flyout.js

```diff
@@ -344,11 +376,41 @@ class EditJobFlyoutUI extends Component {
       );
     }
 
-    return (
+    if (this.state.isConfirmationModalVisible) {
+      confirmationModal = (
+        <EuiOverlayMask>
+          <EuiConfirmModal
```

Comment:
Looking at this example on the https://elastic.github.io/eui/#/guidelines/modals page

![image](https://user-images.githubusercontent.com/7405507/66640351-b6ae0300-ec10-11e9-9668-00f32ca70e9d.png)

 I think we should switch the buttons around to 'Save changes' and 'Leave anyway'. The filled, primary button directs the user to the preferred choice of saving their changes before closing the flyout.

The title of the modal could then be `Save changes before leaving?`

and the text could be `If you don't save, your edits to custom URLs will be lost.`

Reply:
Changed in https://github.com/elastic/kibana/pull/47926/commits/dea29290f199f71947d31120e7fbf8b693c44e47